packages/scripts/package.json

```diff
@@ -42,15 +42,15 @@
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.7.0",
     "enzyme-to-json": "^3.0.0",
-    "eslint": "^4.0.0",
-    "eslint-config-airbnb": "^11.1.0",
-    "eslint-plugin-import": "^1.16.0",
-    "eslint-plugin-jsx-a11y": "^2.2.2",
-    "eslint-plugin-react": "^6.3.0",
+    "eslint": "^5.3.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
+    "eslint-plugin-react": "^7.12.4",
     "file-loader": "^2.0.0",
     "html-webpack-plugin": "3.2.0",
     "jest": "^24.0.0",
-    "jest-environment-jsdom": "^24.0.0",
+    "jest-environment-jsdom": "^23.0.0",
```

Comment:
Why is there a downgrade here ?

Reply:
Oh yeah I fixed the dependencies warnings too.
`jest-environment-jsdom-global` that is used as testing env, is requesting `jest-environment-jsdom` 23